index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="CSS/style.css">
+  </head>
+  <body>
+    <header class="container-header tool-flex">
+      <div class="logo">
+        <a id="my-logo">RolandM</a>
+      </div>
+      <div class="menu">
+        <nav class="nav-bar">
+          <div class="bar-menu">
+            <img src="images/menu.png" id="show-menu" alt="union">
+          </div>
+        </nav>
+      </div>
+    </header>
+    <div class="main-headline">
+      <div class="img-background">
+        <img src="images/background-mobile.png" alt="mobile" id="background">
+      </div>
+      <section class="container-headline b-flex">
```

Comment:
The image is overflowing out of the container. Please add the background image to this section.